rahuldkjain/github-profile-readme-generator
# fixing warning in local environment

## What type of PR is this? (check all applicable)


## Description
A warning is coming up in the local environment.
> Warning: An update to Header inside a test was not wrapped in act(...).
When testing, code that causes React state updates should be wrapped into act(...):
act(() => {
  /* fire events that update state */
});
/* assert on the output */
## Related Tickets & Documents
- commit fixes [#225 ](https://github.com/rahuldkjain/github-profile-readme-generator/issues/225)
## QA Instructions, Screenshots, Recordings
- Run the code locally
- You wont see following warning

<img width="653" alt="Screen Shot 2020-10-07 at 1 36 25 AM" src="https://user-images.githubusercontent.com/46445441/95257447-c2d99a00-083d-11eb-8261-28bb177f2a66.png">



## Added to documentation?
